insights/parsers/dirsrv_logs.py

```diff
@@ -0,0 +1,95 @@
+'''
+389 Directory Server logs
+=========================
+
+'''
+
+from insights import parser, LogFileOutput
+
+
+@parser('dirsrv_access')
```

Comment:
Dangit!  How did that ever work?  We need something to check that all specs defined in decorators correspond to specs in the spec file :-)

OK, I've submitted an update for this and have fixed the 1.x branch in #173